### PR TITLE
Programmatic Tagging

### PR DESCRIPTION
**Pull request by supercid**

### CODE REVIEW

File: app/code/community/Nosto/Tagging/Block/Category.php [hunk 1]
```diff
@@ -49,17 +49,7 @@ class Nosto_Tagging_Block_Category extends Mage_Core_Block_Template
      */
     protected function _toHtml()
     {
-        /** @var Nosto_Tagging_Helper_Account $helper */
```
Comment (mridang) [inline app/code/community/Nosto/Tagging/Block/Category.php]: Why is this removed?

File: app/code/community/Nosto/Tagging/Block/Searchterm.php [hunk 2]
```diff
@@ -42,16 +42,7 @@ class Nosto_Tagging_Block_Searchterm extends Mage_Core_Block_Template
      */
     protected function _toHtml()
     {
-        /** @var Nosto_Tagging_Helper_Account $helper */
-        $helper = Mage::helper('nosto_tagging/account');
```
Comment (mridang) [inline app/code/community/Nosto/Tagging/Block/Searchterm.php]: Why is this removed?

File: app/code/community/Nosto/Tagging/Block/Category.php (hunk 1 above)
Comment (supercid) [reply app/code/community/Nosto/Tagging/Block/Category.php]: It shouldn't. Fixed now, thanks

File: app/code/community/Nosto/Tagging/Block/Searchterm.php (hunk 2 above)
Comment (supercid) [reply app/code/community/Nosto/Tagging/Block/Searchterm.php]: It shouldn't. Fixed now, thanks

**Comment by supercid** [comment]
Closes #404

**Comment by supercid** [comment]
Closes #270
